app/frontend/shared/types/User.res

```diff
@@ -24,10 +24,10 @@ let findById = (id, proxies) =>
   proxies |> ArrayUtils.unsafeFind(proxy => proxy.id == id, "Unable to find a User with ID " ++ id)
 
 let make = (~id, ~name, ~avatarUrl, ~title) => {
-  id: id,
-  name: name,
-  avatarUrl: avatarUrl,
-  title: title,
+  id,
+  name,
+  avatarUrl,
+  title,
```

Comment:
Your rescript plugin is out, Use the latest 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/14979190/188755734-93b1a7f8-a47d-4fdb-8b6a-4d0c42349683.png">